src/sys/signal.rs

```diff
@@ -680,6 +680,8 @@ pub fn kill<T: Into<Option<Signal>>>(pid: ::unistd::Pid, signal: T) -> Result<()
     Errno::result(res).map(drop)
 }
 
+/// Send a signal to a process group [(see
+/// killpg(3))](http://pubs.opengroup.org/onlinepubs/9699919799/functions/killpg.html).
```

Comment:
The docs should say what the arguments mean.  It's not obvious why `signal` should be an `Option`.  Also, I see from the man page that `killpg` does something special when `pgrp` is 0.  Perhaps that should be an option too?